### PR TITLE
remove skip_ from upload_docs (see #835)

### PR DESCRIPTION
**Pull request by svenfuchs**
Can be tested using:

```yaml
deploy:
  provider: pypi
  edge:
    branch: sf-pypi-upload-docs
⋮
```

### CODE REVIEW

File: spec/dpl/providers/pypi_spec.rb [hunk 1]
```diff
@@ -37,16 +37,16 @@
     it { should have_run 'python setup.py other' }
   end
 
-  describe 'given --no-skip_upload_docs' do
-    it { should have_run 'python setup.py upload_docs --upload-dir build/docs -r https://upload.pypi.org/legacy/' }
+  describe 'given --skip_existing' do
```
Comment (webknjaz) [inline spec/dpl/providers/pypi_spec.rb]: This doesn't look right/related to docs...

Comment (webknjaz) [reply spec/dpl/providers/pypi_spec.rb]: Oh, you reordered the tests...

Comment (svenfuchs) [reply spec/dpl/providers/pypi_spec.rb]: oops, yes. sorry, i guess i did :)

**Comment by svenfuchs** [comment]
thanks for the review @webknjaz!

**Pull request merged**